build/templates/_library_interpreter.py/_get_error_description.py.mako

```diff
@@ -0,0 +1,21 @@
+    def get_error_description(self, error_code):
+        '''get_error_description
+
+        Returns the error description.
+        '''
+        try:
+            _, error_string = self.get_error()
```

Comment:
The comment at that link says "Since another thread has changed the status, fall back to the static message lookup." That's not a concern here, since python will use the same thread to call back down to the (local) driver.